scripts/wormhole-local.sh

```diff
@@ -48,7 +50,7 @@ register_client() {
 spawn_wormhole() {
   token=$1
 
-  FLY_TOKEN=$token FLY_PORT=$PORT $GOPATH/src/github.com/superfly/wormhole/bin/wormhole &
+  FLY_TOKEN=$token FLY_PORT=$TLS_PORT $GOPATH/src/github.com/superfly/wormhole/bin/wormhole &
```

Comment:
I haven't seen `TLS_PORT` set anywhere else, is that a typo?

Reply:
It's set. https://github.com/doodles526/wormhole/blob/ad05cd32e20add994deee15250b10a2e368ef4ba/scripts/wormhole-local.sh#L33 and https://github.com/doodles526/wormhole/blob/ad05cd32e20add994deee15250b10a2e368ef4ba/scripts/wormhole-local.sh#L18